examples/mockups/index.html

```diff
@@ -0,0 +1,328 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="/dist/css/site.css" />
+    <script src="/dist/js/vendor.js"></script>
+    <script src="/dist/js/site.js"></script>
+  </head>
+  <body>
+
+    <!-- Top Menu 
+    ----------------------------------- -->
```

Comment:
Also, super nitpick, but it probably helps to keep commenting more minimal. Stylized comments like this throw off editors and indenting, and the rest of the team doesn't stylize comments in our source, so conformance with styles like this will be sporadic at best.